docs/source/app_developers_guide/docker_test_network.rst

```diff
@@ -3,124 +3,174 @@
    :align: center
    :alt: Docker: Sawtooth network with five nodes
 
-Like the single-node environment, this environment uses parallel transaction
-processing and static peering. However, it has the following differences:
+.. include:: ../_includes/about-nw-each-node-runs.inc
+
+Like the :doc:`single-node test environment <docker>`, this environment uses
+parallel transaction processing and static peering.
 
-* PoET simulator consensus instead of dev mode, because dev mode's random-leader
-  consensus is not recommended for multi-node or production networks. Sawtooth
-  offers two versions of :term:`PoET consensus`. PoET-SGX relies on Intel
-  Software Guard Extensions (SGX) to implement a leader-election lottery system.
-  PoET simulator provides the same consensus algorithm on an SGX simulator.
 
-* An additional transaction processor, PoET Validator Registry, handles PoET
-  settings for a multiple-node network.
+.. _prereqs-multi-docker-label:
 
 Prerequisites
 -------------
 
-This procedure assumes that you have already created a single-node environment,
-as described in :doc:`docker`. Refer to the previous procedure for more
-information on each step.
+* This application development environment requires Docker Engine and Docker
+  Compose.
 
-If the single-node environment is still running, shut it down. Enter CTRL-c from
-the window where you originally ran ``docker-compose up``, then run the
-following command from your host system:
+  * Windows: Install the latest version of
+    `Docker Engine for Windows <https://docs.docker.com/docker-for-windows/install/>`_
+    (also installs Docker Compose).
 
-.. code-block:: console
+  * macOS: Install the latest version of
+    `Docker Engine for macOS <https://docs.docker.com/docker-for-mac/install/>`_
+    (also installs Docker Compose).
 
-   $ docker-compose -f sawtooth-default.yaml down
+  * Linux: Install the latest versions of
+    `Docker Engine <https://docs.docker.com/engine/installation/linux/ubuntu>`_
+    and
+    `Docker Compose <https://docs.docker.com/compose/install/#install-compose>`_.
+    Then follow
+    `Post-Install steps
+    <https://docs.docker.com/install/linux/linux-postinstall/#manage-docker-as-a-non-root-user>`_.
 
-For more information, see :ref:`stop-sawtooth-docker-label`.
+* If you created a :doc:`single-node Docker environment <docker>` that is
+  still running, shut it down and delete the existing blockchain data and logs.
+  For more information, see :ref:`stop-sawtooth-docker-label`.
 
 
 Step 1: Download the Docker Compose File
 ----------------------------------------
 
-Download the Docker Compose file for a multiple-node network,
-`sawtooth-default-poet.yaml <./sawtooth-default-poet.yaml>`_.
-Save this file in the same directory as the single-node compose file
-(``sawtooth-default.yaml``).
+Download the Docker Compose file for a multiple-node network.
 
+* For PBFT: Download
+  `sawtooth-default-pbft.yaml <./sawtooth-default-poet.yaml>`_
```

Comment:
Link is to poet dockerfile.

You'll also need to add a step to the Makefile to copy the compose file to the correct spot. Here's how the poet compose file is copied: https://github.com/hyperledger/sawtooth-core/blob/master/docs/Makefile#L71

Reply:
I fixed the borked link for the sawtooth-default-pbft.yaml file. Thanks for catching this silly error.

For the Makefile change: I assumed that this had already been done in the PR that added the PBFT yaml file.  It appears not.  Let's do this in a separate PR.

Reply:
Created a separate commit for the docs/Makefile.